strr-host-pm-web/app/stores/hostApplication.ts

```diff
@@ -12,6 +12,7 @@ export const useHostApplicationStore = defineStore('host/application', () => {
     agreedToRentalAct: z.boolean().refine(val => val, { message: t('validation.required') }),
     agreedToSubmit: z.boolean().refine(val => val, { message: t('validation.required') })
   })
+  const { isRenewal, registrationId } = useRouterParams()
```

Comment:
As mentioned in the chat, I don't like the idea of using query params to get renewal status or reg id. For time being and testing, we can stick with it, but I will work on a new solution in a new PR.